selfservice/flow/login/persistence.go

```diff
@@ -18,7 +18,8 @@ type (
 	RequestPersister interface {
 		CreateLoginRequest(context.Context, *Request) error
 		GetLoginRequest(context.Context, uuid.UUID) (*Request, error)
-		UpdateLoginRequest(context.Context, uuid.UUID, identity.CredentialsType, *RequestMethod) error
+		UpdateLoginRequestMethod(context.Context, uuid.UUID, identity.CredentialsType, *RequestMethod) error
+		UpdateLoginRequestReauth(ctx context.Context, id uuid.UUID, reauth bool) error
```

Comment:
Since this is a one-way method (reauth is false by default and can only be set to true) wouldn't it make sense to name this something like `MarkLoginRequestForced`? Alternatively `UpdateLoginRequest(context.Context, *Request)`.